build-pipeline/vg-pipeline-module-pr.yml

```diff
@@ -158,19 +174,27 @@ stages:
             sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
             sudo chmod +x /usr/local/bin/docker-compose
             cd vanguard-api
-            mv .env.docker .env
             cat .env
             echo pull request branch name $(System.PullRequest.SourceBranch)
             go env -w GO111MODULE=on
             go env -w GOPRIVATE=github.com/cldcvr/*
             skip_dev=1 make init
+            git status
+            make docker-pull
```

Comment:
is this actually needed? Shouldn't docker-run pull the containers if necessary?

Reply:
right, I think I'll still need to check. I have to try. this is internal docker behaviour, on local I've observed that since the images for my platform (arm) are not available on DH, it would build. but if I run make docker-pull it pulls the images away. I added it earlier since I observed that the images were not getting pulled and so the pipeline was taking more time in build. later found it was because of .env.docker change. so I still need to revert this line and check.

Reply:
it prefers to build by default it seems: https://dev.azure.com/cldcvrdevusa/Vanguard/_build/results?buildId=47251&view=logs&j=f833b1df-30e0-5a1f-cb52-9403a4b80eb5&t=76b0bae2-bfd1-5756-d2a1-e8c237efe5eb